### DISCUSSION TITLE
Error: Invalid or corrupt jarfile acp_commander.jar

### DISCUSSION THREAD

**Discussion by bbotts77** [1000001101000/acp-commander#2] [Q&A]
Windows 10
Java(TM) SE Runtime Environment (build 1.8.0_291-b10)

When I attempt to run the current build of acp-commander.jar, that's the error I get. Any idea what I'm doing wrong?

**Comment by 1000001101000** [comment]
Good to hear from you!

It probably means something went wrong with the download. Sometimes folks download the html of the download page instead of the binary by accident.

Take a look at the file and see if the size seems wrong, or take a look with wordpad and see if it’s full of html.

**Reply by bbotts77** [reply]
That's exactly what I did. Thanks for reminding me how github works. LOL

**Reply by 1000001101000** [reply]
I've done the same thing a few times. 

On the bright side with a jar it just won't work, when it happens with boot images it can be a pain to recover from.

**Reply by bbotts77** [reply]
When you pointed out that I probably downloaded the web page as the jar file, I realized I had also done that with the images. So, I have downloaded those actual files and compared the sizes to make sure I had the right stuff. Thanks again. That's what I get for trying to be productive in the middle of the night.

**Reply by 1000001101000** [reply]
lol

I’d say the best lesson I learned from years as a casual Linux user is not to start an install past a certain time of night.  When I was in my 20’s I think it was midnight.... it’s more like 9pm now.
